strawberry/extensions/add_validation_rule.py

```diff
@@ -0,0 +1,15 @@
+from typing import Type
+
+from graphql import ASTValidationRule
+
+from strawberry.extensions.base_extension import Extension
+
+
+def AddValidationRule(validation_rule: Type[ASTValidationRule]):
+    class _AddValidationRule(Extension):
+        def on_request_start(self):
+            self.execution_context.validation_rules = (
+                self.execution_context.validation_rules + (validation_rule,)
+            )
+
+    return _AddValidationRule
```

Comment:
I think this should accept multiple validation rules 😊

Reply:
Yep can do